src/Environment.php

```diff
@@ -804,6 +807,8 @@ public function mergeGlobals(array $context): array
 
     /**
      * @internal
+     *
+     * @return array<string, array{precedence: int, class: class-string}>
```

Comment:
could be `class-string<AbstractUnary>` to be more precise (same for binary operators but using `AbstractBinary`).

Reply:
Done